heroku_deploy.sh

```diff
@@ -1,5 +1,12 @@
 #!/bin/bash
-echo "Deploying to Heroku"
-docker tag todo-app:prod registry.heroku.com/todo-test-application/web
-docker push registry.heroku.com/todo-test-application/web
-heroku container:release web -a todo-test-application
\ No newline at end of file
+if [[ "$TRAVIS_BRANCH" == "master" ]]; then
+  echo "Deploying app to Heroku";
+  echo "Tagging heroku docker image";
+  docker tag $DOCKER_USERNAME/todo-app registry.heroku.com/todo-test-application/web;
```

Comment:
It would be nice to pull the application name (`todo-test-application`) out into an environment variable, and also to mention on your README that the site is now accessible at http://todo-test-application.herokuapp.com/

Reply:
updated